apps/web/src/screens/Profile/modals/components/ModalHeader.tsx

```diff
@@ -0,0 +1,35 @@
+import { FC } from 'react';
+import { modalActions, ModalConfigs } from '@walless/app';
+import { Times } from '@walless/icons';
+import { Stack, Text } from '@walless/ui';
+
+interface Props {
+	content: string;
+	config: ModalConfigs;
+}
+
+const ModalHeader: FC<Props> = ({ content, config }) => {
+	return (
+		<Stack
+			display="flex"
+			flexDirection="row"
+			justifyContent="space-between"
+			alignItems="center"
+			width="100%"
+		>
+			<Text color="#FFFFFF" fontWeight="500" fontSize={20}>
+				{content}
+			</Text>
+			<Stack
+				onPress={() => {
+					modalActions.destroy(config.id);
+				}}
+				cursor="pointer"
+			>
+				<Times size={16} />
+			</Stack>
```

Comment:
why not use `Button` here? Beside `cursor` that indicate clickable button, the feedback on interaction like reducing opacity is also required to provide a better user experience